### PR TITLE
More advanced L2 integration tests

### PR DESCRIPTION
**Pull request by ajsantander**
Instructions
1. `git clone git@github.com:ethereum-optimism/optimism-integration.git`
2. `cd optimism-integration`
3. `docker-compose pull`
4. `./up.sh`
5. `cd ~/synthetix` (or wherever)
6. `node publish deploy-ovm-pair`
7. `npm run test:prox:ovm`

### CODE REVIEW

File: test/optimism/index.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+	before('set up providers', () => {
+		this.providerL1 = new ethers.providers.JsonRpcProvider('http://localhost:9545');
+		this.providerL2 = new ethers.providers.JsonRpcProvider('http://localhost:8545');
+	});
+
+	before('set up signers', () => {
+		// owner
+		// See publish/src/commands/deploy-ovm-pair.js
+		ownerAddress = '0x640e7cc27b750144ED08bA09515F3416A988B6a3';
```
Comment (justinjmoses) [inline test/optimism/index.js:36]: could we at least share this between the files using some kind of constant for the PK?

File: test/optimism/withdrawals.test.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+		let SynthetixL2, SynthetixBridgeToBaseL2, IssuerL2;
+
+		// --------------------------
+		// Setup
+		// --------------------------
+
+		before('identify signers', async () => {
+			// See publish/src/commands/deploy-ovm-pair
+			user1L2 = new ethers.Wallet(
+				'0x5b1c2653250e5c580dcb4e51c2944455e144c57ebd6a0645bd359d2e69ca0f0c',
```
Comment (justinjmoses) [inline test/optimism/withdrawals.test.js:23]: as before

**Review by justinjmoses — APPROVED** [review]
Great stuff Ale - just a request to share those PKs in a common file somewhere to de-dupe them

File: test/optimism/index.js (hunk 1 above)
Comment (ajsantander) [reply test/optimism/index.js]: Done

File: test/optimism/withdrawals.test.js (hunk 2 above)
Comment (ajsantander) [reply test/optimism/withdrawals.test.js]: Done

**Pull request merged**